SmurfTestingScripts/README.md

```diff
@@ -2,34 +2,34 @@
 Python script used for automated StateSmurf testing.
```

Comment:
Maybe look at the CodeRabbit flow image... It will be really nice to have something similar as part of the documentation.